SourceGitphp/SourceGitphp.php

```diff
@@ -0,0 +1,367 @@
+<?php
+
+# Copyright (c) 2012 John Reese
+# Licensed under the MIT license
+
+if ( false === include_once( config_get( 'plugin_path' ) . 'Source/MantisSourcePlugin.class.php' ) ) {
+	return;
+}
+
+require_once( config_get( 'core_path' ) . 'url_api.php' );
+
+class SourceGitphpPlugin extends MantisSourcePlugin {
+	public function register() {
+		$this->name = plugin_lang_get( 'title' );
+		$this->description = plugin_lang_get( 'description' );
+
+		$this->version = '0.18';
```

Comment:
You probably want to set your own version number instead of reusing the one from SourceGitweb.